ReactWindows/ReactNative.Shared/Bridge/IReactContext.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System;
+
+namespace ReactNative.Bridge
+{
+    /// <summary>
+    /// Context wrapper for the React instance to manage lifecycle events.
+    /// </summary>
+    public interface IReactContext : IDisposable

```

Comment:
I get the spirit of what you're trying to do here, I just don't see the utility of it, and eliminating this interface would definitely reduce the diff for this PR.

---
In reply to: [80748748](https://github.com/ReactWindows/react-native-windows/pull/713#discussion_r80748748) [](ancestors = 80748748)